app/portainer/components/datatables/registries-datatable/registriesDatatable.html

```diff
@@ -62,10 +62,10 @@
                 <a ui-sref="portainer.registries.registry.access({id: item.Id})" ng-if="$ctrl.accessManagement">
                   <i class="fa fa-users" aria-hidden="true"></i> Manage access
                 </a>
-                <a ui-sref="portainer.registries.registry.repositories({id: item.Id})" ng-if="$ctrl.registryManagement" class="space-left">
+                <a ui-sref="portainer.registries.registry.repositories({id: item.Id})" ng-if="$ctrl.registryManagement && item.URL !== 'quay.io'" class="space-left">
                   <i class="fa fa-search" aria-hidden="true"></i> Browse
                 </a>
-                <a ui-sref="portainer.extensions.extension({id: 1})" ng-if="!$ctrl.registryManagement" class="space-left" style="color: #767676" tooltip-append-to-body="true" tooltip-placement="bottom" tooltip-class="portainer-tooltip" uib-tooltip="Feature available via an extension">
+                <a ui-sref="portainer.extensions.extension({id: 1})" ng-if="!$ctrl.registryManagement && item.URL !== 'quay.io'" class="space-left" style="color: #767676" tooltip-append-to-body="true" tooltip-placement="bottom" tooltip-class="portainer-tooltip" uib-tooltip="Feature available via an extension">
```

Comment:
This is a dirty hack.
Such check must be made in the controller, and bool variable be introduced to the view.
Perhaps even add a constant list of registries that should not be browsed, not just single one, but that is optional for the future.

But  **if**  inside view is a dirty hack.

Reply:
>Perhaps even add a constant list of registries that should not be browsed, not just single one

Thanks for pointing that !
Although, ng-if is not a dirty hack, it's how angular works for conditional display.